charts/node/templates/statefulset.yaml

```
@@ -743,6 +785,12 @@ spec:
       {{- toYaml . | nindent 8 }}
       {{- end }}
       volumes:
+      {{- range $keys := .Values.node.extraConfigmapMounts }}
```

The $keys variable is unused here, you can replace by;

```suggestion
      {{- range .Values.node.extraConfigmapMounts }}
```